composer.json

```diff
@@ -80,6 +80,7 @@
     "extra": {
         "branch-alias": {
             "dev-master": "1.x-dev",
+            "dev-incremental-parallel": "1.x-dev",
```

Comment:
Please remember to remove this before you're done. 

Optional: You could also do me a favor and remove dev-state at the same time. 😛